tests/test_hll_serialization.py

```diff
@@ -8,11 +8,58 @@
 from copy import deepcopy
 from python_hll.hlltype import HLLType
 from python_hll.hll import HLL
+from python_hll.serialization import SerializationUtil
+from python_hll.util import BitUtil
 
 # A fixed random seed so that this test is reproducible.
 RANDOM_SEED = 1
 
 
+def test_parameters_byte():
+    assert 5 == SerializationUtil.register_width(0x85)
+    assert 5 == SerializationUtil.register_count_log2(0x85)
+
+    assert 0x85 == SerializationUtil.pack_parameters_byte(5, 5)
```

Comment:
If I try this using the Java code in https://github.com/aggregateknowledge/java-hll I get -123:

```
package net.agkn.hll;

import net.agkn.hll.serialization.SerializationUtil;

public class JonTest {

    public static void main(String[] args) {
        System.out.println(SerializationUtil.packParametersByte(5, 5));
    }
    
}
```

Result: -123

Reply:
So the source of my confusion is that apparently `NumberUtil.to_hex` will convert both 133 and -123 to 0x85, so it should work either way. I think it would be better to have `to_bytes` always return non-negative output since then we could use python's`binascii.hexlify(bytearray(hll.to_bytes()))` instead of `NumberUtil.to_hex`, but I understand not wanting to make that change so feel free to close this PR